api/Element.json

```diff
@@ -2868,7 +2868,7 @@
               }
             ],
             "ie": {
-              "version_added": null
+              "version_added": false
```

Comment:
`Document` has a "MSFullscreenChange" altname for IE11. Can you check if that event was fired on just `Document` or on `Element` and bubbling? If it was fired on elements then this should change.

Reply:
As far as I can tell, there appears to be no Element-level support for the fullscreen API in IE.  The Document API defines an `onmsfullscreenchange` handler, but the Element API does not.

Reply:
OK, that's enough of a clue, no testing needed!